.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+If this is a general issue, you are encouraged to use the following guideline.
+
+Please make sure your issue is about crouton rather than software
```

Comment:
[This comment](https://github.com/dnschneid/crouton/issues/262#issuecomment-21058391) suggests that it's better to create an issue about software running in the chroot so we can determine whether it is or isn't due to the crouton environment.  So I don't think this sentence is needed.

Instead, I'd stick to just the bottom part with the template headers, perhaps tweaked to be closer to the [wiki page](https://github.com/dnschneid/crouton/wiki/Common-issues-and-reporting).